### PR TITLE
F#10 more blocks

### PR DESCRIPTION
**Pull request by kirstyellis**
pushing the things that work for now

### CODE REVIEW

File: generators/python/pose.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  var text_z = block.getFieldValue('z');
+
+  var code = "";
+  code += "wait = \"" + dropdown_wait.toString() + "\"\n";
+  code += "x_displacement = " + text_x.toString() + "\n";
+  code += "y_displacement = " + text_y.toString() + "\n";
+  code += "z_displacement = " + text_z.toString() + "\n";
+
+
+  function readPythonFile(file)
```
Comment (toliver) [inline generators/python/pose.js]: Could this function be declared somewhere else and imported?

**Comment by kirstyellis** [comment]
I moved the function 'readpythonfile' to utils, is that cool?

File: blocks/initialise.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Blockly.Blocks['initialise'] = {
+  init: function() {
+    this.appendDummyInput()
+        .appendField("Load Commander")
+        .appendField(new Blockly.FieldDropdown([["Arm Commander", "arm_commander"], ["Hand Commander", "hand_commander"], ["Hand and Arm", "hand_and_arm"]]), "commander")
```
Comment (ugocupcic) [inline blocks/initialise.js]: they could use 2 blocks for `hand_and_arm` no?

File: core/utils.js [hunk 3]
```diff
@@ -577,6 +577,24 @@ Blockly.genUid = function() {
   return id.join('');
 };
 
+Blockly.readPythonFile = function(file) {
```
Comment (ugocupcic) [inline core/utils.js:580]: :+1:

**Pull request merged**